### ISSUE TITLE
icon with a pencil where is nothing to edit (articles)

### ISSUE THREAD

**Issue by JensB74** [contao/contao#5926]
### Affected version(s)

all

### Description

Different customers called me, that they have clicked on a pencil and can't change anything there.

This pencil on the left side is it:
![pencils](https://user-images.githubusercontent.com/32700129/229751270-9a4f8987-abcd-4281-a863-d70705124f8b.jpg)
the pencil opens a new window with a preview of the page, perhaps an other icon would make more sense here ..

**Comment by bezin** [comment]
I cannot reproduce this in the online demo. 

* You mention articles in the issue title, but the screenshot is from the page tree. Does it happen in both modules?
* Do you refer to the first pencil icon (edit page) or the last icon (edit article)?

**Comment by fritzmg** [comment]
They mean the page type icon, which opens the preview of the page, which also has a pencil in it.

**Comment by JensB74** [comment]
I now logged in into the online demo to check it there, in article view all of the left Icons with the pencil go to a new window, previewing that page/article instead of editing it, with was misunderstood for our customers, see this screenshot:
![edit-icons](https://user-images.githubusercontent.com/32700129/230018393-a3d8191d-d7cb-4b59-b68e-e6811cb64d78.jpg)

perhaps an Icon like this one would make more sense
![browser](https://user-images.githubusercontent.com/32700129/230018850-f5a7889e-e7e1-4599-9d38-cd1379d17385.png)
<a href="https://www.flaticon.com/free-icons/internet" title="internet icons">Internet icons created by Freepik - Flaticon</a>

**Comment by fritzmg** [comment]
An eye icon is already used for the published/invisible state/action.

**Comment by JensB74** [comment]
it was just a suggestion ;) we also have several pencils at the moment ;)

**Comment by fritzmg** [comment]
Yes, but isn't your point to not use several Icons with a pencil in it?

**Comment by JensB74** [comment]
I just wanted to point out that there is a pencil in a place where there is nothing to edit, but a link to a preview. My icon was just a suggestion, of course any icon can be used, which makes sense for the devs, here's another suggestion, without an eye
![browser2](https://user-images.githubusercontent.com/32700129/230022278-9dbe2ec6-2d40-4257-a96f-7553952f6598.png)
<a href="https://www.flaticon.com/free-icons/seo-and-web" title="seo and web icons">Seo and web icons created by Prosymbols - Flaticon</a>

I don't want to sell icons here, just draw attention to the grievance ;)

**Comment by JensB74** [comment]
und mal kurz auf deutsch, ich weiß echt nicht, was immer da Problem ist, man weißt hier auf ein kleines Problem hin, was ein Anwender hatte und es wird tot zerredet, ist mir jetzt schon mehrfach aufgefallen, so macht es echt keine Freude, solche Sachen noch mitzuteilen, tut mir leid ... ich fand auch, dass der Kunde da einen Punkt hatte, wenn ich ihm sage, klick auf das Stift-Symbol und er klickt auf der erste Stiftsymbol was er findet und landet dann auf der Vorschau, statt in der Bearbeitung des Artikels, dass es verwirrend ist .. und ob da jetzt schon ein Auge existiert oder zwei Stifte, das ist nicht mein Problem, es war nur ein Hinweis, Icons könnt ihr euch natürlich so aussuchen wie ihr wollt. Einen schönen Tag noch ..

**Comment by fkaminski** [comment]
Maybe just as is but without pencil (The help Info is furthermore self explained)

![image](https://user-images.githubusercontent.com/7679799/230023110-f1f8777a-3a26-4fdb-8d7f-9f1a78568d6b.png)
![image](https://user-images.githubusercontent.com/7679799/230023214-049918e0-cc4a-41e7-b37f-b62799914854.png)

**Comment by fritzmg** [comment]
> ich fand auch, dass der Kunde da einen Punkt hatte

I agree as well (at least I can see how it could be confusing). I was merely pointing out that changing it to an eye might yield the same issue again.

**Comment by asaage** [comment]
I agree with @fkaminski without pencil is better since there is currently no preview icon.
If there was such an icon we could use it also in the list of preview-links.
Furthermore i find it a bit strange that a preview is clickable in articles and pages but not in news, events and not at all in **"preview links"**.
Having that in articles is debateable in my opinion (i rarely see the urls like pagealias/articles/articlealias out there).
Also the title of that link (tooltip) should probably be "**PRE**View in a new window".
A common icon would be something like that i guess: 
![image](https://user-images.githubusercontent.com/4138336/230055692-386344ad-56dd-4912-8744-fb44a287d6b1.png)

**Comment by fritzmg** [comment]
For the _page_ icon I assume the pencil is there to denote that it's a regular page, i.e. a page with _editable_ content. In contrast to a redirect page (arrow) for example.

**Comment by asaage** [comment]
> to denote that it's a regular page, i.e. a page with editable content.

true but - the link encapsulating the icon does not edit the page/article as visual suggested nor does it link to the publicly available version as the tooltip suggests. 🙈🤷‍♂️🤣
Maybe we should have any actions aligned on the right side then.

**Comment by fritzmg** [comment]
> nor does it link to the publicly available version as the tooltip suggests

But it does?

**Comment by asaage** [comment]
> But it does?

kind of...
it's a preview wich potentialy shows only fragments of a page (in case you clicked on an article) and hidden things if that is selected in the preview-bar.
I had cases where such urls like https://demo.contao.org/preview.php/en/accordion.html were used in public newsletters or facebook-posts.

**Comment by fritzmg** [comment]
> I had cases where such urls like https://demo.contao.org/preview.php/en/accordion.html were used in public newsletters or facebook-posts.

But that's unrelated to this discussion and also unrelated to your statement

**Comment by JensB74** [comment]
Because the Eye-Icon is already in use, perhaps the devs could change it to something like this (without texts "online/offline"):
![online-offline](https://user-images.githubusercontent.com/32700129/230344862-f712f89f-6aa6-400f-9afb-cc60bcc6a7d4.jpg)
(Image is Screenshot from the CMS Redaxo)
and use the Eye-Icon for the "Preview-Page" then.

**Comment by fritzmg** [comment]
> and use the Eye-Icon for the "Preview-Page" then.

The "preview page" icon is actually the icon for that specific type of page. Currently we have the following page type icons:

* <img src="https://demo.contao.org/system/themes/flexible/icons/regular.svg">
* <img src="https://demo.contao.org/system/themes/flexible/icons/logout.svg">
* <img src="https://demo.contao.org/system/themes/flexible/icons/forward.svg">
* <img src="https://demo.contao.org/system/themes/flexible/icons/redirect.svg">
* <img src="https://demo.contao.org/system/themes/flexible/icons/feed.svg">
* <img src="https://demo.contao.org/system/themes/flexible/icons/error_401.svg">

Plus different variants for when the page is hidden in the menu or protected or unpublished.

An eye icon would not be suitable as the icon should still denote the page's type. Instead of a pencil it should probably be something that visually resembles "content".

**Comment by ausi** [comment]
> Instead of a pencil it should probably be something that visually resembles "content".

Agree, maybe three lines that symbolize text?

**Comment by fritzmg** [comment]
Yes, or maybe even add a tiny rectangle to resemble an image - but that icon may be too small for that 😁

**Comment by leofeyer** [comment]
It should also be colored somehow, otherwise it would look quite monotonous:

<img width="316" alt="" src="https://user-images.githubusercontent.com/1192057/230364235-4ea72be8-6eb1-4b41-aef1-a57faf0f6fe0.png">

<img width="367" alt="" src="https://user-images.githubusercontent.com/1192057/230364269-c3275de5-084f-4f29-b763-35941f6bccf8.png">

**Comment by fritzmg** [comment]
Green might look nice: ![regular](https://user-images.githubusercontent.com/4970961/230375076-39b11601-255b-4339-b485-23bb2a3cb9b5.svg)

**Comment by fkaminski** [comment]
I like that (pure) way - JM2C

**Comment by JensB74** [comment]
that icon doesn't encourage me to 'click it', what's missing, in my opinion, is something like a magnifying glass or an eye as invitation

**Comment by fritzmg** [comment]
> that icon doesn't encourage me to 'click it', what's missing, in my opinion, is something like a magnifying glass or an eye as invitation

You mean like the crossed eye for a page that is hidden in the menu? I think that would be too much. imho this functionality only needs to be documented and does not need a visual queue as well.

**Comment by JensB74** [comment]
here's another suggestion, I put it together from several individual elements, maybe it's a bit too detailed and needs to be simplified
![icons](https://user-images.githubusercontent.com/32700129/230379459-4aab9305-3808-44d8-8d4d-b8dec9bd4b91.jpg)
Icons are from:
https://www.freepik.com/free-vector/seo-search-engine-optimization-doodle-icons_22139998.htm#query=magnifier%20icon&position=9&from_view=search&track=ais

**Comment by fkaminski** [comment]
Too much in my opinion.
>  ...needs to be simplified

Already suggested above (sometimes less is more) :-)

**Comment by fkaminski** [comment]
> It should also be colored somehow

Maybe with just a color touch in the upper right corner for the regular page (base) symbol. The article symbol than as proposed in https://github.com/contao/contao/issues/5926#issuecomment-1498917730

![image](https://user-images.githubusercontent.com/7679799/230556312-37e8377b-ac09-423a-aab9-0f13bf4d649c.png)


> ... that icon doesn't encourage me to 'click it'

Cursor & tooltip sufficiently point to the corresponding behavior

**Comment by Toflar** [comment]
As discussed on today's call, this is one of many issues concerning icons. We're planning going back to an icon set. Once we have that, we can all discuss which icon of a predefined icon set is the best. This should simplify the task a lot 😄

**Comment by zoglo** [comment]
Could be closed 🤔 ?

**Comment by fritzmg** [comment]
Yes, as of Contao 5.4 a new icon set is in use where there is no pencil icons anymore on actions where you actually do not edit anything.
